web/webkit/src/main/scala/net/liftweb/http/Req.scala

```diff
@@ -284,14 +284,14 @@ FileParamHolder(name, mimeType, fileName)
  * @param localFile The local copy of the uploaded file
  */
 class OnDiskFileParamHolder(override val name: String, override val mimeType: String,
-                            override val fileName: String, val localFile: File) extends
+                            override val fileName: String, val localFile: Path) extends
```

Comment:
See https://github.com/lift/framework/pull/1852#issuecomment-302056802 . Folks are using `.localFile` directly, and passing it to things that expect a `File`. Moreover, `localFile` is a poor name for something that isn't a `File`, but rather a `Path`, IMO.

I'll work up a PR to make this API-compatible, should only be a couple of lines.